### PR TITLE
remove redirect leading to 443

### PR DESCRIPTION
**Pull request by jburel**
this is currently leading to error on latest CI. Removing the redirect

### CODE REVIEW

**Comment by jburel** [comment]
ci is back to normal. So we can either include or close

File: omero/developers/index.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ with commercial licenses and customization available from |Glencoe|. You can
 read about how OMERO has developed since the project started in the
 :doc:`/users/history`.
 
-.. _GNU General Public License (GPL): https://www.gnu.org/copyleft/gpl.html
+.. _GNU General Public License (GPL): https://www.gnu.org/licenses/gpl-3.0.html
```
Comment (sbesson) [inline omero/developers/index.rst]: An alternative would be
```suggestion
.. _GNU General Public License (GPL): https://www.gnu.org/licenses/licenses.html#GPL
```

As this link is specific to GPL 3 while some of the code is under GPL 2

Comment (jburel) [reply omero/developers/index.rst]: This will work.
We did not see the error again on latest-ci

**Review by sbesson — APPROVED** [review]
Assuming CI is passing

**Pull request merged**